Bio/GenBank/Scanner.py

```diff
@@ -1393,6 +1393,14 @@ def _feed_header_lines(self, consumer, lines):
                             data += ' ' + line[GENBANK_INDENT:]
                         else:
                             # We now have all the data for this entry:
+
+                            # The DEFINITION field must ends with a period
+                            # # see ftp://ftp.ncbi.nih.gov/genbank/gbrel.txt [3.4.5]
+                            # and discussion https://github.com/biopython/biopython/pull/616
+                            # We consider this period belong to the syntax, not to the data
+                            # So remove it if it exist
+                            if line_type == 'DEFINITION' and data.endswith('.'):
+                                data = data.rstrip('.')
```

Comment:
That would remove multiple trailing dots, e.g. ``"etc...".rstrip(".") == "etc"``